slam-libraries/Makefile

```diff
@@ -1,7 +1,14 @@
 BUILD_CHANNEL?=local
 
 bufsetup:
-	sudo apt-get install -y protobuf-compiler-grpc libgrpc-dev libgrpc++-dev || brew install grpc openssl --quiet
+ifeq ("Darwin", "$(shell uname -s)")
+	sudo rm /usr/local/bin/go
```

Comment:
This was necessary to unlink go version 1.17, but I don't love having to do this run running `bufsetup` locally.